backend/src/server.ts

```diff
@@ -0,0 +1,38 @@
+import {appConfig} from './config/app-config';
+import {NestFactory} from '@nestjs/core';
+import {AppModule} from './app.module';
+import * as bodyParser from 'body-parser';
+import {ValidationPipe} from './common/pipes/validation.pipe';
+
+export function main() {
+
+  appConfig.setAppConfig(process.argv[2] || 'local');
```

Comment:
Maybe using env is a better choice than command line arguments?
It's easier to integrate with docker and other deployment platforms.

Reply:
perfect